docs/en/02_Developer_Guides/14_Files/07_File_Usage.md

```diff
@@ -0,0 +1,84 @@
+---
+title: File usage
+summary: See file usage and customising the file "Used on" table
+icon: compress-arrows-alt
+---
+
+# File Usage
+
+CMS users can view where a file is used by accessing the "Used On" tab in the file editor. This feature allows them to identify what DataObjects depend on the File.
+
+In the Files section of the CMS, click on a file to see file details.  Within the file details panel there is a
+"Used on" tab that shows a table of Pages and other DataObjects where the file is used throughout the website.
+
+## Customising the File "Used on" table in the Files section (asset-admin)
+
+Your project specific DataObject will automatically be displayed on the "Used on". This may not always be desirable, especially when working with background DataObjects the user can not interact with directly. Extensions can be applied
```

Comment:
```suggestion
Your project specific DataObject will automatically be displayed on the Used on tab. This may not always be desirable, especially when working with background DataObjects the user can not interact with directly. Extensions can be applied
```